sentry/src/main/java/io/sentry/SentryMeasurementUnit.java

```diff
@@ -0,0 +1,40 @@
+package io.sentry;
+
+import java.util.Locale;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * <a href="https://getsentry.github.io/relay/relay_metrics/enum.DurationUnit.html">Develop Docs</a>
+ */
+public enum SentryMeasurementUnit {
```

Comment:
`h`: MeasurementUnit are actually 3 different enums in Python; see 
https://github.com/getsentry/sentry-python/blob/37e165edd633bfde5927150633193bc1bf41eab1/sentry_sdk/_types.py#L53-L82. Relay calls them MetricUnit and they kind of align with Python, see https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html

I know that we already implemented this like that in Dart, https://github.com/getsentry/sentry-dart/pull/1011, but maybe we should reconsider before moving forward.

We could define an interface for `SentryMeasurementUnit` and create different enums implementing the interface for duration, information, fraction, custom. 

Or we could create inner enums inside a class
```JAVA
public class SentryMeasurementUnit2 {
  public static enum Duration {
    SECOND, MINUTE
  }

  public enum Information {

  }
}
```

Reply:
Agree, see my comment https://github.com/getsentry/sentry-java/pull/2260#discussion_r983365457. @marandaneto are you open to changing it for dart?

Reply:
Apparently, I only checked the [linked relay source code](https://github.com/getsentry/relay/blob/ed3a521f773ecd4bf9a2aefd7af80080d56d0841/relay-common/src/constants.rs#L353-L374) and didn't see the other ones, we have to fix this.

Reply:
I will address this here https://github.com/getsentry/sentry-dart/pull/1033